.github/workflows/cloudbeat-ci.yml

```diff
@@ -34,6 +34,14 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
 
+      - name: Fetch OPA version
```

Comment:
LGTM, if u can please add these steps to  the  nightly ci job as well
https://github.com/elastic/cloudbeat/blob/main/.github/workflows/Periodic-CI.yml

Reply:
@DaveSys911 so we have to duplicate?
Isn't there a way to have these steps once and consume them from both jobs?

Reply:
@eyalkraft we have a task for it, but currently it does.